DateTimePicker is missing after @nextcloud/vue is upgraded to v3

### Steps to reproduce
1. Upgrade to the latest git-commit of nextcloud-calendar
2. Try to modify an existing event or create a new event

### Expected behaviour
I can choose the date & time as in a5ed1b6f and older versions

![圖片](https://user-images.githubusercontent.com/1937689/120010248-80084480-c00f-11eb-9b14-9d95bdce5868.png)

### Actual behaviour
The date/time picker below the title is missing

![圖片](https://user-images.githubusercontent.com/1937689/120010124-5b13d180-c00f-11eb-8b57-1c7c6983da6a.png)

### Calendar app
**Calendar app version:** 396f811c (first bad commit indicated by `git bisect`)

**CalDAV-clients used:** (Thunderbird Lightning, DAVx5, Evolution, macOS Calendar, etc)

### Client configuration
**Browser:** Firefox 88.0.1

**Operating system:** Arch Linux

### Server configuration

(I believe this is a client issue. Please tell me if server information is really needed)

### Logs

#### Web server error log (e.g. /var/log/apache)
#### Log file (data/nextcloud.log)

#### Browser log
```
No OC found index.js:46:12
JQMIGRATE: Migrate is installed, version 3.3.2 jquery-migrate.min.js:2:708
jQuery is deprecated: The global jQuery is deprecated. It will be updated to v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own. globals.js:60:15
$ is deprecated: The global jQuery is deprecated. It will be updated to v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own. globals.js:60:15
[INFO] calendar: 
Initial settings:
	- AppVersion: 2.2.0
	- EventLimit: true
	- FirstRun: false
	- ShowWeekNumbers: false
	- ShowTasks: true
	- ShowWeekends: true
	- SkipPopover: true
	- SlotDuration: 00:30:00
	- DefaultReminder: none
	- TalkEnabled: false
	- TasksEnabled: true
	- Timezone: automatic
 
Object { app: "calendar", uid: "yen" }
ConsoleLogger.js:46:10
[DEBUG] calendar: The calendar app is using version 2.2019c of the timezone database 
Object { app: "calendar", uid: "yen" }
ConsoleLogger.js:42:10
Fallback to locale en moment.js:83:11
[INFO] calendar: Updated moment locale: nextcloud-calendar-fake-locale-en-zh-tw 
Object { app: "calendar", uid: "yen" }
ConsoleLogger.js:46:10
session heartbeat polling started session-heartbeat.js:97:9
[DEBUG] unified-search: Unified Search initialized with the following providers 
Object { 0: {…}, 1: {…}, 2: {…}, 3: {…}, 4: {…}, 5: {…}, app: "unified-search", uid: "yen" }
ConsoleLogger.js:42:18
[DEBUG] calendar: Current user principal is /remote.php/dav/principals/users/yen/ 
Object { app: "calendar", uid: "yen" }
ConsoleLogger.js:42:10
[ERROR] calendar: t is null 
Object { app: "calendar", uid: "yen" }
ConsoleLogger.js:54:10
[ERROR] calendar: t is null 
Object { app: "calendar", uid: "yen" }
ConsoleLogger.js:54:10
NONE RepeatFreqInterval.vue:63
```

--- a/src/components/Shared/CalendarPicker.vue
+++ b/src/components/Shared/CalendarPicker.vue
@@ -6,10 +6,10 @@
 		:options="calendars"
 		:value="calendar"
 		@select="change">
-		<template slot="singleLabel" slot-scope="scope">
+		<template v-slot:singleLabel="scope">
 			<CalendarPickerOption v-bind="scope.option" />
 		</template>
-		<template slot="option" slot-scope="scope">
+		<template v-slot:option="scope">
 			<CalendarPickerOption v-bind="scope.option" />
 		</template>
 	</Multiselect>
@@ -64,3 +64,9 @@ export default {
 	},
 }
 </script>
+
+<style lang="scss" scoped>
+::v-deep .multiselect__tags {
+	margin: 3px 0;
+}
+</style>

--- a/src/components/Shared/DatePicker.vue
+++ b/src/components/Shared/DatePicker.vue
@@ -40,8 +40,7 @@
 		@close="close"
 		@change="change"
 		@pick="pickDate">
-		<template
-			slot="icon-calendar">
+		<template v-slot:icon-calendar>
 			<button
 				class="datetime-picker-inline-icon icon"
 				:class="{'icon-timezone': !isAllDay, 'icon-new-calendar': isAllDay, 'datetime-picker-inline-icon--highlighted': highlightTimezone}"
@@ -63,7 +62,7 @@
 		</template>
 		<template
 			v-if="!isAllDay"
-			slot="footer">
+			v-slot:footer>
 			<button
 				v-if="!showTimePanel"
 				class="mx-btn mx-btn-text"

--- a/src/views/EditSidebar.vue
+++ b/src/views/EditSidebar.vue
@@ -46,32 +46,6 @@
 			</EmptyContent>
 		</template>
 
-		<template
-			v-if="!isLoading && !isError"
-			v-slot:primary-actions>
-			<PropertyCalendarPicker
-				v-if="showCalendarPicker"
-				:calendars="calendars"
-				:calendar="selectedCalendar"
-				:is-read-only="isReadOnly"
-				@selectCalendar="changeCalendar" />
-
-			<PropertyTitleTimePicker
-				:start-date="startDate"
-				:start-timezone="startTimezone"
-				:end-date="endDate"
-				:end-timezone="endTimezone"
-				:is-all-day="isAllDay"
-				:is-read-only="isReadOnly"
-				:can-modify-all-day="canModifyAllDay"
-				:user-timezone="currentUserTimezone"
-				@updateStartDate="updateStartDate"
-				@updateStartTimezone="updateStartTimezone"
-				@updateEndDate="updateEndDate"
-				@updateEndTimezone="updateEndTimezone"
-				@toggleAllDay="toggleAllDay" />
-		</template>
-
 		<template v-slot:header>
 			<IllustrationHeader :color="illustrationColor" :illustration-url="backgroundImage" />
 		</template>
@@ -93,6 +67,32 @@
 			<ActionButton v-if="canDelete && canCreateRecurrenceException" icon="icon-delete" @click="deleteAndLeave(true)">
 				{{ $t('calendar', 'Delete this and all future') }}
 			</ActionButton>
+		</template>
+
+		<template
+			v-if="!isLoading && !isError"
+			v-slot:description>
+			<PropertyCalendarPicker
+				v-if="showCalendarPicker"
+				:calendars="calendars"
+				:calendar="selectedCalendar"
+				:is-read-only="isReadOnly"
+				@selectCalendar="changeCalendar" />
+
+			<PropertyTitleTimePicker
+				:start-date="startDate"
+				:start-timezone="startTimezone"
+				:end-date="endDate"
+				:end-timezone="endTimezone"
+				:is-all-day="isAllDay"
+				:is-read-only="isReadOnly"
+				:can-modify-all-day="canModifyAllDay"
+				:user-timezone="currentUserTimezone"
+				@updateStartDate="updateStartDate"
+				@updateStartTimezone="updateStartTimezone"
+				@updateEndDate="updateEndDate"
+				@updateEndTimezone="updateEndTimezone"
+				@toggleAllDay="toggleAllDay" />
 		</template>
 
 		<AppSidebarTab
@@ -368,3 +368,9 @@ export default {
 	},
 }
 </script>
+
+<style lang="scss" scoped>
+::v-deep .app-sidebar-header__description {
+	flex-direction: column;
+}
+</style>

